src/CDFAttribute.php

```diff
@@ -52,17 +52,15 @@ class CDFAttribute {
    *
    * @throws \Exception Unsupported data type exception.
    */
-  public function __construct($id, $type, $value = NULL, $language = 'und') {
+  public function __construct($id, $type, $value, $language = 'und') {
```

Comment:
This whole file needs to stay the same, so it shouldn't be included in this diff.